.github/workflows/build_makefile_test.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Test
```

Comment:
How about renaming this file to `build_test_makefile.yml`? And for CMake, `build_test_cmake.yml`

Reply:
I agree. I would change it.